src/core/environment.go

```diff
@@ -561,19 +561,64 @@ func diskPartitions() (partitions []*proto.DiskPartition) {
 }
 
 func releaseInfo() (release *proto.ReleaseInfo) {
-	hostInfo, err := host.Info()
+	const osReleaseFile =  "/etc/os-release"
```

Comment:
are there any other paths we should try? https://www.linux.org/docs/man5/os-release.html says it might be at `/etc/os-release` or `/usr/lib/os-release`

Reply:
I did read about os-release path in this link earlier.

But as I was looking into `amplify-nginx-agent` implementation, it falls back to use [release](https://github.com/nginxinc/nginx-amplify-agent/blob/fa4fb352ee6db134899364f5c8a3e0f1bd536ed3/amplify/agent/common/util/host.py#L127) file from `/etc` path depending upon system.

Hence I happen to use only `/etc/os-release` [as implemented in  `amplify-nginx-agent` ]